src/ray/python/test_driver_taskchains.py

```diff
@@ -0,0 +1,105 @@
+import argparse
```

Comment:
For new tests, we should use `pytest` (as opposed to `unittest`). Are these tests being parsed by some unit testing framework?

Reply:
@stephanie-wang